src/Markdown.MAML/Model/MAML/MamlCommand.cs

```diff
@@ -31,6 +31,8 @@ public List<MamlParameter> Parameters
 
         public bool SupportCommonParameters { get; set; }
 
+        public string ModuleName { get; set; }
```

Comment:
There are cases when it's `null`.
I.e. when the function is dynamically defined, we should handle this cases gracefully.